src/utils/findGitmojiCommand.ts

```diff
@@ -1,19 +1,35 @@
-// @flow
-import COMMIT_MODES from '@constants/commit'
+import { CommitModes } from '@constants/commit'
 import FLAGS from '@constants/flags'
 
-const isSupportedCommand = (command: ?string, options: Object): boolean => {
-  return Object.keys(options).includes(command)
+type Options = {
+  [key: string]: object
+}
```

Comment:
```suggestion
type Options = Record<string, any>;
```

Reply:
Actually no, would probably be better if you imported options from `cli.ts` and specified this as `typeof options`.